plugin.rb

```diff
@@ -9,6 +9,7 @@
 
 enabled_site_setting :discourse_reactions_enabled
 
+register_asset "stylesheets/mixins/scrollbar.scss"
```

Comment:
where is this used?

Reply:
Oops. I removed these and need to remove here as well.